spec/cb/requests/cover_letter/update_spec.rb

```diff
@@ -0,0 +1,82 @@
+# Copyright 2015 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+require 'spec_helper'
+
+module Cb
+  module Requests
+    module CoverLetter
+      describe Update do
+        let(:cover_letter_update_request) { described_class.new args }
+
+        let(:args) do
+          {
+            external_id: external_id,
+            external_user_id: external_user_id,
+            cover_letter_title: cover_letter_title,
+            text: text,
+            test: test
+          }
+        end
+
+        let(:external_id) { 'external id' }
+        let(:external_user_id) { 'external user id' }
+        let(:cover_letter_title) { 'cover letter title' }
+        let(:text) { 'text' }
+        let(:test) { true }
+
+        it { expect(described_class).to be < Cb::Requests::Base }
```

Comment:
Wow, you're blowing my mind a little bit. Had no idea you could assert the inheritance chain like that.